tools/adev-api-extraction/BUILD.bazel

```diff
@@ -0,0 +1,70 @@
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+load("@aspect_rules_esbuild//esbuild:defs.bzl", "esbuild")
+load("//tools:defaults.bzl", "ts_project")
+load("@aspect_rules_ts//ts:defs.bzl", rules_js_tsconfig = "ts_config")
+
+package(default_visibility = ["//visibility:public"])
+
+esbuild(
```

Comment:
I assume bundling would not be needed anymore when you use` js_binary` from `@aspect_rules_js//js:defs.bzl`